pavelib/utils/test/suites/nose_suite.py

```diff
@@ -190,7 +190,8 @@ def _default_test_id(self):
 
         if self.root == 'lms':
             default_test_id += " {system}/tests.py"
-            default_test_id += " openedx/core/djangolib"
+            default_test_id += " openedx/core/djangolib/*"
+            default_test_id += " openedx/features/*"
```

Comment:
I made the change for consistency (and see the comment above that recommends it). If that is causing the breakage then it is fine to change it back. Let's double check the coverage once we get a clean build so we can be sure that all the tests are running.